README.md

```diff
@@ -1,9 +1,16 @@
 # Bacon
 Bacon is a new Java CLI for PNC 2.0 combining features of old PNC, DA CLI and PiG tooling.
 
+## Documentation
+The [Developer's guide](https://github.com/project-ncl/bacon/blob/master/DEVELOPING.md) is a good start on how to get up and running with developing in Bacon.
+
+The [PiG build-config.yaml guide](https://github.com/project-ncl/bacon/wiki/Anatomy-of-a-build-config.yaml) provides information on how to build your own build-config.yaml file for PiG.
```

Comment:
Did you want to write PiG `build-config.yaml` (with backticks) to match the other usage?

Reply:
Not sure if I can do that in a link. Will check. You're also not supposed to be on Github this week Mr @dwalluck

Reply:
That worked! Done.